docs/SMCWG-charter.md

```diff
@@ -0,0 +1,328 @@
+assurance” is to be determined and defined by this SMCWG through studying the
+existing methods that exist in the industry, as well as identity management
+frameworks and any applicable legislation.
+
+An S/MIME certificate can also be used in an automated message with transfer
+agents that use cryptographic security services that do not require any human
+intervention, such as the signing of software-generated documents and the
+encryption of fax messages sent over the Internet. While these existing use
+cases are not in scope for the SMCWG, care will be exercised by the SMCWG to
+avoid unintended adverse effects to these uses. The security, stability and
+resiliency of the Internet shall be taken into consideration when the SMCWG
+forms its consensus. The SMCWG will consult other technical communities when and
+as necessary.
+
+The problem to be addressed by the working group is the absence of consistent
+and audited validation practices used by CAs in establishing the identity of the
+subject (if present) and verifying that the subscriber controls the email
+address. While there are methods for validating control of a domain that can be
+adopted from TLS certificates, there are currently no standard requirements for
+validating control of email addresses. Methods for validating identities in TLS
+certificates also exist and should be leveraged where possible, as well as other
+identity validation standards common in the industry.
+At least one method to efficiently validate an email address as well as
+developing a consistent profile for S/MIME certificates in order to facilitate
+technical interoperability across the Internet are also to be addressed by
+the SMCWG.
+
+## Chartering of the S/MIME Certificate Working Group
+
+A Chartered Working Group (“SMCWG”) is created to perform the activities as
+specified in this Charter, subject to the terms and conditions of the
+[CA/Browser Forum Bylaws](https://cabforum.org/bylaws/) and [Intellectual
+Property Rights (IPR) Policy](https://cabforum.org/ipr-policy/), as such
+documents may change from time to time. This charter for the S/MIME Certificate
+Working Group has been created according to CAB Forum Bylaw 5.3.1. In the event
+of a conflict between this Charter and any provision in either the Bylaws or
+the IPR Policy, the provision in the Bylaws or IPR Policy SHALL take
+precedence. The definitions found in the Forum’s Bylaws SHALL apply to
+capitalized terms in this Charter.
+
+### 1. Scope
+
+The authorized scope of the SMCWG SHALL be to discuss, adopt, and maintain
+policies, frameworks, and sets of standards related to the issuance and
+management of S/MIME certificates by third-party CAs under a publicly trusted
+root, limited as follows:
+
+- Baseline verification of control over email addresses, including those used
+  by a natural person or a legal entity, or used by automated systems such as
+  for mailing lists
+- Identity validation for natural persons and legal entities
```

Comment:
As noted in https://cabforum.org/pipermail/public/2020-March/014917.html , one of the conclusions from the Cupertino and Thessaloniki F2F was that if identity is included in scope of the charter, that it's gated as a deliverable within the scope of the charter.

To that end, I think that we can remove this bullet, and then add something around line 94 that says:

---
Following the production of guidelines that meet these objectives, the SMCWG MAY further consider:
- Identity validation for natural persons and legal entities
---

Reply:
There was a later conversation in Guangzhou in which Google (IMO) stated that it would not support the inclusion of identity in the scope of the charter at all, even if deferred to a later phase. Has that position changed? Does the proposed MAY imply that any member could veto work on identity validation?

Reply:
@wthayer No, the proposed MAY does not imply that. It just simply doesn't commit the SMCWG to tackle that as the immediate next item, if new priorities come up.

I also don't think that's an accurate summary re: Guangzhou. Identity within the scope of the charter is certainly problematic for participation. This is because the inclusion in scope creates IP challenges and risks, and resolving those can take considerable time, preventing some members from joining or being able to contribute usefully to the core discussions unrelated to identity.

However, if identity is scoped as a fair topic from the get-go, which has been repeatedly agreed as not a goal, simply eliminates any hope of there being a useful or timely deliverable of core value, as shown by both the chartering of this WG and the SCWG experiences. If that's the case, it's likely valuable to not bother at all participating, and as a consequence of that lack of participation, not adopt any guidelines or audit criteria that might be produced.

Reply:
I disagree that "identity is scoped as a fair topic from the get-go" has been "repeatedly agreed as not a goal". I have certainly indicated a willingness to compromise on this point, but have also heard from other members who were less willing, leading me to the conclusion that the only resolution is to ballot it. I can support either approach, so I'll leave it to others to decide which route to take with the (new to me) knowledge that Google may be willing to participate initially if identity is explicitly scoped to a later phase of work.

Reply:
I'm sure I lack a substantial amount of context, and while I would prefer identity validation be a focus of the WG only after guidelines regarding email validation and certificate profiles are produced (as the above proposed change accomplishes), this is an area I'm in agreement with Wayne on in that "identity" is an inclusion I'm willing to compromise on.

Reply:
Thanks Clint and Wayne.  Let me reiterate that we support validation and profiles being the first priority, what we can't support is the exclusion of discussion of important topics from charter scope, especially given how long and difficult future charter updates will inevitably and unfortunately be.  No one is being asked to vote for or support any requirements, only to allow the topic to be discussed.  And the main reason we want to make it clear it can be discussed is because of a concerted effort to silence those who think it's an important topic.  The speed at which the group can accomplish its first priorities will depend on the willingness of participants to work together and dedicate the time and effort necessary to write a quality document; I am completely unconvinced that efforts to exclude important subjects from the WG scope will in any way help with that.  I'm very thankful for Apple, Microsoft and Mozilla's recent and substantial contributions to this charter, and look forward to actually being able to start productive work on minimum requirements validation and profiles for S/MIME certificates very soon.

Reply:
@timfromdigicert Yes, the minutes from our meetings show there have been three options, basically since the very first time we were discussing:

1. Exclude from charter entirely, recharter after first deliverable (of core e-mail validation & profile)
2. Include in charter, gated upon delivering the first deliverable
3. Include in charter, without any limitations

The main distinction between 1 and 2 are the IP commitments and review that is entailed in producing work. The main distinction between 2 and 3 is making sure that we don't get bogged down in endless discussions about identity without having made some progress on the core deliverable.

(1) was Google's proposal, because that represents the broadest possible charter to allow collaboration. This is important, especially as we've seen how past SCWG chartering efforts have prevented a number of members to participate.

(2) was DigiCert's proposal, as reflected in the minutes. This attempts to avoid needing to recharter, despite this being "just" a Forum ballot to adopt language such as shown in https://github.com/sleevi/cabforum-docs/pull/9 , but otherwise scopes the SMCWG's activities to solving the core problems first.

(3) is what this charter is.

https://github.com/sleevi/cabforum-docs/pull/9 attempts to do (2). As noted, this creates more barriers for collaboration, and the value provided by avoiding a recharter is potentially greatly overstated. The primary benefit of (1) vs (2) is it allows more participation, quicker, by more members, but it's clear that's not a shared value. However, (3) is very much a non-starter, because without the limits (such as in (2)), there's a clear set of interests that prevent a useful deliverable in a timely fashion.

Reply:
If someone wants to try to obstruct the work of the committee, there are plenty of options available to do regardless of what's in scope or not.  I'm much less concerned that's an actual issue, as I haven't seen or heard any evidence either publicly or privately that indicate anyone is planning to do so, and it's a pretty serious accusation.  Furthermore, in the event it happens, the best recourse is the Code of Conduct, which explicitly forbids such activities.

Reply:
@timfromdigicert I'm sorry you felt this is a "pretty serious accusation", especially since I don't know what "this" is you're referring to.

The minutes show that there is active interest in tackling identity as a co-equal, or even primary, deliverable among CA members. I'm happy to point out specific instances, if you'd like public examples. Such an approach would not be a violation of the Code of Conduct, because it would be entirely within the chartered scope. I think you may have interpreted (3) as suggesting some malicious intent, but nothing of the sort: merely, good faith attempts at debating identity will, and clearly have for two years, impeded progress on a charter.

That's exactly why a charter like (2) is essential to setting forth priorities. If there is not agreement on a consistent set of priorities, and the ballot fails, then that's a useful and important signal. The entire point of the chartering process is to help lay out those baseline priorities, so that it's clear if and when something does become a Code of Conduct problem.

This is why (1) or (2) are the only possible charters we can support, and (2) is likely a charter that will prevent us from joining for some time. (3) is a charter that we can neither join nor support, because without that prioritization, it is an unwise and unproductive use of our time.